jdk-1.6-parent/select2-parent/select2/src/main/java/org/wicketstuff/select2/AbstractSelect2Choice.java

```diff
@@ -137,19 +146,21 @@ public AbstractSelect2Choice(String id, IModel<M> model, List<T> collection)
 	 *            renderer list item
 	 * @see HiddenField#HiddenField(String, IModel)
 	 */
+	// will be dropped in 7.0
+	@Deprecated
 	public AbstractSelect2Choice(String id, IModel<M> model, List<T> choices,
-		ChoiceRenderer<T> renderer)
+	                             ChoiceRenderer<T> renderer)

```

Comment:
not sure why change like this is necessary, seems to be redundant
can you remove it? (lots in this file)